cs-many-resources/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Should this be in a root folder?  I don't think it has been standard to include this `.gitignore` elsewhere?